terraform/dev/main.tf

```diff
@@ -30,16 +30,16 @@ module "gke" {
   subnetwork = module.vpc.gke_subnetwork
 
   gke_nodepools = [
-    {
-      auto_repair = true
-      auto_upgrade = false
-      min_node_count = 1
-      max_node_count = 10
-      machine_type = "n1-standard-2"
-      disk_size_gb = "50"
-      preemptible = false
-      version = "1.14.6-gke.1"
-    },
+//    {
```

Comment:
TF supports `/* .. */` for multi-line comments.  Also, we should just put a note in here that says "Scaling down dev cluster to 0 while not needed" or something to that effect.

Reply:
Done